packages/admin/blocks-admin/src/blocks/helpers/composeBlocks/createSettingsBlock.ts

```diff
@@ -6,13 +6,15 @@ interface Options<State> {
     AdminComponent: BlockAdminComponent<State>;
     isValid?: (values: State) => boolean | Promise<boolean>;
     definesOwnPadding?: boolean;
+    extractTextContent?: boolean;
```

Comment:
This enabled the feature for all settings, although we might need it only for specific ones.

pro: easier to use (single boolean)

con: might not be always what we want

possible workaround: split into two blocks, one extractTextContent enabled, one disabled

Reply:
The feature needs to be explicitly enabled, is this really a problem? Or is this about accidentally enabling it for non-text settings, e.g. a select?